doc/sphinx-guides/source/api/client-libraries.rst

```diff
@@ -10,11 +10,18 @@ Python
 
 https://github.com/IQSS/dataverse-client-python is the offical Python package for Dataverse APIs.
 
-`Robert Liebowitz <https://github.com/rliebz>`_ from the `Center for Open Science <http://centerforopenscience.org>`_ heads its development and the library is used to integrate the `Open Science Framework (OSF) <http://osf.io>`_ with Dataverse via an add-on which itself is open source and listed on the :doc:`/api/apps` page.
+`Robert Liebowitz <https://github.com/rliebz>`_ created this library while at the `Center for Open Science (COS) <http://centerforopenscience.org>`_ and the COS uses it to integrate the `Open Science Framework (OSF) <http://osf.io>`_ with Dataverse via an add-on which itself is open source and listed on the :doc:`/api/apps` page.
```

Comment:
Small nitpick: `http => https` Both http://centerforopenscience.org and http://osf.io will redirect to the `https` versions.  Other than that, LGTM!